compile.sh

```diff
@@ -23,5 +24,12 @@ if [ -d out ]; then
   cp spec/* out/
   rm out/{README.md,Makefile,*.py,*.bs}
 
+  echo 'Copying samples/* -> out/samples/'
+  cp samples/* out/samples/
+
+  echo 'Copying tools/* -> out/tools/'
```

Comment:
Hm, seems like today `tools` is just spec tools, not things like the workload simulator. Perhaps we could just put the workload simulator in samples/ and omit `tools` from the deployment - WDYT?

Reply:
Well, tools would be ok too. Up to you. The extglob below is kind of arcane though, is there something simpler?

Reply:
Yeah, it's kind of arcane. I'm fine with putting it in `samples/` instead and that will avoid needing the weird glob.